### PR TITLE
chore: Added python module

### PR DESCRIPTION
**Pull request by ssepml**
## Description

Added basic functionality to read codes from `python`

### CODE REVIEW

**Review by MalotteAtKivra — APPROVED** [review]

**Review by efcasado-kivra — APPROVED** [review]

**Review by moritzploss-k — COMMENTED** [review]
can we add one basic test and make it part of the CI flow (call `from_code` with one given error code and validate that the output is correct)?

Also, I think it's useful to have the `APIError` as a class so that one can do something like this:
```python3
err = KivraAPIError.from_code('40104')
print(err.status_code)
print(err.payload)
```
but we can also add that later

File: kivra_api_errors.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        global errors
+        return errors.get(code)
+
+    @staticmethod
+    def is_code(code) -> bool:
+        global errors
+        if errors.get(code):
+            return True
+
+        return False
```
Comment (moritzploss-k) [inline kivra_api_errors.py:30]:
```suggestion
        return code in errors
```

Comment (ssepml) [reply kivra_api_errors.py]: Done

**Comment by ssepml** [comment]
> can we add one basic test and make it part of the CI flow (call `from_code` with one given error code and validate that the output is correct)?
> 
> Also, I think it's useful to have the `APIError` as a class so that one can do something like this:
> 
> ```python
> err = KivraAPIError.from_code('40104')
> print(err.status_code)
> print(err.payload)
> ```
> 
> but we can also add that later

Added test and included in GitHub workflow

**Review by moritzploss-k — APPROVED** [review]

**Pull request merged**